src/table/data-grid-focus.directive.ts

```diff
@@ -57,7 +57,9 @@ export class DataGridFocus {
 		switch (event.key) {
 			case "Right": // IE specific value
 			case "ArrowRight":
-				if (element.nextElementSibling && Array.from(headerRow).indexOf(element.nextElementSibling) < headerRow.length - 1) {
+				const firstBodyRow = rows[1].querySelectorAll("td");
+				if (element.nextElementSibling && Array.from(headerRow).indexOf(element.nextElementSibling) < firstBodyRow.length) {
+					event.preventDefault();
```

Comment:
This might be a UX question. We might want to prevent default regardless of if we're taking action. Does the spec say something about that?